compel/src/lib/infect.c

```diff
@@ -186,6 +192,29 @@ static int skip_sigstop(int pid, int nr_signals)
 	return 0;
 }
 
+#define SIG_MASK(sig) (1ULL << ((sig)-1))
+
+#define SIG_IN_MASK(sig, mask) ((sig) > 0 && (sig) < SIGMAX && (SIG_MASK(sig) & (mask)))
```

Comment:
Should not it instead be `<= SIGMAX` as SIGMAX is a valid signal number? See how we iterate in prepare_sigactions_from_image up to sig==SIGMAX.